bundles/org.openhab.binding.wemo/src/main/java/org/openhab/binding/wemo/internal/handler/WemoCoffeeHandler.java

```diff
@@ -200,48 +232,37 @@ public void onValueReceived(@Nullable String variable, @Nullable String value, @
         // We can subscribe to GENA events, but there is no usefull response right now.
     }
 
-    private synchronized void onSubscription() {
-        if (service.isRegistered(this)) {
-            logger.debug("Checking WeMo GENA subscription for '{}'", this);
-
-            String subscription = "deviceevent1";
-            if (subscriptionState.get(subscription) == null) {
-                logger.debug("Setting up GENA subscription {}: Subscribing to service {}...", getUDN(), subscription);
-                service.addSubscription(this, subscription, SUBSCRIPTION_DURATION_SECONDS);
-                subscriptionState.put(subscription, true);
+    private synchronized void addSubscription() {
+        synchronized (upnpLock) {
+            if (service.isRegistered(this)) {
+                logger.debug("Checking WeMo GENA subscription for '{}'", this);
```

Comment:
Old code, but: Showing in logs like this:
```
Checking WeMo GENA subscription for 'org.openhab.binding.wemo.internal.handler.WemoHandler@12b127f'
```

If you want, could be changed to:
```suggestion
                logger.debug("Checking WeMo GENA subscription for '{}'", getThing().getUID());
```

for being more usable (all handlers).

Reply:
Changed in all handler classes.

Reply:
Reopening comment as it seems you agreed to change, but actually it hasn't been changed.

Reply:
Changed leftovers.